CMakeLists.txt

```diff
@@ -18,7 +18,44 @@ include (cmake/target.cmake)
 include (cmake/tools.cmake)
 include (cmake/ccache.cmake)
 include (cmake/clang_tidy.cmake)
-include (cmake/git_status.cmake)
```

Comment:
Why don't we just add these changes to the `.cmake` file and not bring it into the main `CMakeLists.txt`?

Reply:
Done